_docs/reference/gui.md

```diff
@@ -7,4 +7,20 @@ module: reference/gui
 
 # Graphical user interface (GUI)
 
-This feature is coming soon.
+StorageOS provides a GUI for cluster and volume management.
+
+**Note**: This feature is currently in alpha. To access the GUI from version 0.10.0,
+set the environment variable `ALPHA_FEATURES=true`.
```

Comment:
`ALPHA_FEATURES=true` was only needed for 0.9.  In 0.10 it's enabled by default.  Maybe change to say that it's in beta?

Reply:
Done.